cmd/dgo/build.go

```diff
@@ -144,12 +144,16 @@ func PerformBuild(cmd *cobra.Command, args []string, cmdArguments *BuildCmdArgum
 		return pkgError
 	}
 
-	if cmdArguments.docker && !tools.IsDocker() {
-		logrus.Infof("Building docker container")
-		err := tools.Exec(cmd.Context(), curDir, []string{"docker", "build", "--build-arg", fmt.Sprintf("%s=true", SkipBuildEnv), "."}, env)
-		if err != nil {
-			logrus.Errorf("Failed to build docker container %v", err)
-			return err
+	for _, root := range args {
```

Comment:
Probable it would be good to check if Docker files for particular applications are exists, or it is only one main docker file.